src/NoFrixion.MoneyMoov/Models/Payruns/UpdatePayrunInvoiceAmount.cs

```diff
@@ -0,0 +1,21 @@
+// -----------------------------------------------------------------------------
+// Filename: UpdatePayrunInvoice.cs
+// 
+// Description: Contains details to update the invoice's amount to pay request
+//
+// Author(s):
+// Pablo Maldonado (pablo@nofrixion.com)
+// 
+// History:
+//  26 Jan 2024  Pablo Maldonado   Created, Harcourt Street, Dublin, Ireland.
+// 
+// License:
+// MIT.
+// -----------------------------------------------------------------------------
+
+namespace NoFrixion.MoneyMoov.Models;
+
+public class UpdatePayrunInvoiceAmount
```

Comment:
We name the update models like PaymentRequestUpdate, PayoutUpdate and so on. I would name this as PayrunInvoiceUpdate.

Reply:
Yep. Will close this PR and create a new one with a different approach where more fields will be able to be updated.

Thanks.